cli/cli/commands/service/add/add.go

```diff
@@ -203,14 +200,6 @@ var ServiceAddCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCo
 			Type:    flags.FlagType_Bool,
 			Default: fullUuidFlagKeyDefault,
 		},
-		{
-			Key: mapPortsFlagKey,
```

Comment:
Unused and undocumented flag.  We decided to not support `service add` with the new `no-connect` flag.

Reply:
1. Should we track a TODO to support this overtime?
2. What is the behavior if i create an enclave with `no-connect` and then do this? Happy for us to address this concern in a later PR

Reply:
With this change, the ports for this newly added service won't be forwarded if the flag `no-connect` was passed to the previous `run` command.

Reply:
> 1. Should we track a TODO to support this overtime?

From the Connect API design doc: "We could build kurtosis service connect, kurtosis port connect, etc. as shorthands for connecting an entire service or port if we need. At the moment, I'm not convinced that it's worth adding the extra surface area/mental load to users."

Reply:
> 2. What is the behavior if i create an enclave with `no-connect` and then do this? Happy for us to address this concern in a later PR

With this change, the ports for this newly added service won't be forwarded if the flag no-connect was passed to the previous run command.